csv-import.mdx

```diff
@@ -0,0 +1,76 @@
+- Importing with the correct data types is important. For example, a column might use numbers in the first few records, but then have a string value further in the file.
+- Treating and reporting errors in a user-friendly way is important. No on wants to attempt importing millions of rows only to be hit with an unhelpful error. 
+
+Keeping these items in mind and taking into account valuable feedback from our user community, we've crafted a more user-friendly approach for data importing and exporting using the Xata UI. This means an easier, more intuitive way to work with your data, saving you time and making the whole process smoother.
+
+As part of our [launch week](link-when-ready), we’re introducing a new CSV import UI experience, built from scratch, to make your Xata journey simpler - allowing you to automatically add new tables by importing your data directly into a Xata database. Also, we’ve added an export feature that allows you seamlessly download your data as a CSV file.
+
+## Preview before you import
+
+To get started click **Import CSV** in the sidebar:
+
+![Import a CSV file from the sidebar](/images/csv-import-sidebar.png)
+
+Browse or drag your CSV file (read [the docs](/docs/csv-data/import-data#format-types) to see the formats we support).
+
+Xata will preview the first 1000 rows in the file and intelligently guess the column types based on the data in your CSV.
+
+In some cases the column types might not be what you want, in which case you can change them in the drop-down and the preview will refresh. You can also rename columns.
+
+![Selecting the column types](/images/csv-import-column-name-type.png)
+
+Errors are highlighted in red. When you’re happy with how the preview looks, you can...
+
+## Start the import
+
+Importing large files can take a while, so be sure to leave your browser tab open while the import happens.
+
+Any errors encountered while importing rows will be displayed in real-time. You'll have the option to download a summary of all these errors once the import is finished.
+
+![Showing the CSV import errors](/images/csv-import-errors.png)
+
+If something isn’t right you can stop the import and go back to the preview screen to tweak your settings.
+
+There is no cap on the size of the files you can upload - we’ve successfully imported 10 million rows in a 1GB CSV file.
+
+## Exporting to CSV
+
+We’ve also added the highly requested feature to download a Xata table as a CSV file.
+
+To download your table click the **Actions** button at the top of your table:
+
+![You can download your data as a CSV file](/images/csv-export-button.png)
+
+There are two options for downloading a CSV file:
+
+- **Export full table to CSV:** This exports all the rows from your table. Links only include the ID. You might find this useful for creating a quick back up, or exporting data into another system (or right back into another Xata table).
```

Comment:
There could be an opportunity to go deeper here with a screenshot of each option, but not really necessary